pages/learning_MDAnalysis.md

```diff
@@ -25,16 +25,32 @@ introduction to the library and there are [other
 tutorials]({{site.github.wiki}}/Tutorials)
 available, too.
 
+[Interactive Jupyter
+notebooks](http://nbviewer.jupyter.org/github/MDAnalysis/binder-notebook/tree/master/notebooks/)
+show how to accomplish specific tasks (including visualizing
+trajectories with nglview); these notebooks can be run in the cloud on
```

Comment:
I suppose we could add a link to `nglview` docs, though probably not crucial

Reply:
done